content/en/cd-as-a-service/tasks/tenants/switch-tenants.md

```diff
@@ -0,0 +1,38 @@
+---
+title: View and Switch Tenants
+description: >
+  View and switch tenants in your Armory CD-as-a-Service organization.
+---
+
+## {{% heading "prereq" %}}
+
+* You have a [custom Tenant Admin role]({{< ref "cd-as-a-service/tasks/iam/create-role#user-role-examples" >}}) that applies to more than one tenant **or** you have the Organization Admin role.
```

Comment:
the placement for this is a bit odd. In context of the page the line makes it seem like, a user needs to be tenant admin to be able to switch tenants. which is not true.

Reply:
Absolutely. Thanks for catching that. Hasty copy/paste. Ooops.